handsontable/test/e2e/mobile/filters.spec.js

```diff
@@ -0,0 +1,43 @@
+const id = 'testContainer';
+
+describe('Filters', () => {
+  beforeEach(function() {
+    this.$container = $(`<div id="${id}"></div>`).appendTo('body');
+    window.resizeTo(390, 840);
+  });
+
+  afterEach(function() {
+    if (this.$container) {
+      destroy();
+      this.$container.remove();
+    }
+  });
+
+  it('should keep focus on the input when the filter search input is clicked', async() => {
+    const hot = handsontable({
+      data: Handsontable.helper.createSpreadsheetObjectData(3, 3),
+      filters: true,
+      colHeaders: true,
+      dropdownMenu: true,
+    });
+
+    const th = hot.view._wt.wtTable.getColumnHeader(0);
+    const button = th.querySelector('.changeType');
+
+    simulateTouch(button);
+
+    const plugin = hot.getPlugin('dropdownMenu');
+    const input = plugin.menu.container.querySelector('.htUIMultipleSelectSearch input');
```

Comment:
If possible, I would use test helpers to make the test content as small as possible. Other than that, I checked the test locally, and it failed on Safari (mobile). Can you check it? I would be for fixing the test so that it passes on all browsers, finally if it is not possible, maybe condition only on Android?
```suggestion
    handsontable({
      data: createSpreadsheetObjectData(3, 3),
      filters: true,
      colHeaders: true,
      dropdownMenu: true,
    });

    const button = getCell(-1, 0).querySelector('.changeType');

    simulateTouch(button);

    const plugin = getPlugin('dropdownMenu');
    const input = plugin.menu.container.querySelector('.htUIMultipleSelectSearch input');
```